web/client/plugins/tutorial/preset/geostory_edit_tutorial.js

```diff
@@ -1,62 +1,26 @@
 /**
- * Copyright 2018, GeoSolutions Sas.
+ * Copyright 2020, GeoSolutions Sas.
```

Comment:
@tdipisa @MV88  In this tutorial there is an action to open the meia  editor.
I think it's a bit complex as action to be included in the current tutorial implementation because it affect also the reducers of mediaEditor and geostory.

```js
{
        translationHTML: "geostoryEditMediaEditor",
        selector: ".ms-geostory",
        position: "auto",
        action: {
            next: {
                type: 'GEOSTORY:ADD',
                id: uuid(),
                path: 'sections',
                position: 'title_section_id1',
                element: {
                    id: uuid(),
                    type: 'paragraph',
                    title: 'Media Section',
                    contents: [
                        {
                            id: uuid(),
                            type: 'column',
                            contents: [
                                {
                                    id: uuid(),
                                    type: 'media',
                                    size: 'medium',
                                    align: 'center'
                                }
                            ]
                        }
                    ]
                }
            }
        }
    }
```

Reply:
@allyoucanmap we are goind to replace the media editor step with one less invasive.
i'm going to add a step that highlights the pencil and shows some info about media editor